cmake/packaging.cmake

```
@@ -160,4 +163,15 @@ cpack_add_component(
 )
 cpack_add_component(ttnn-validation GROUP nn-validation)
 
+cpack_add_component_group(ttml)
+cpack_add_component(
+    ttml
+    DEPENDS
+        nn
+        metalium
+    GROUP ttml
+    DESCRIPTION "TT-Train runtime library"
+)
+cpack_add_component(ttml GROUP ttml)
```

I assume you wouldn't want a package named tt-ttml. Since tt- is automatically prefixed, we just use the stuff after that as our component group name.  We give a Description to a component with the exact same name so the group uses it.  And because we like names like 'ttml' for the other files, we map a component of that name into the 'ml' group (which will get the tt prefix tacked on later).


```suggestion
cpack_add_component_group(ml)
cpack_add_component(
    ml
    DEPENDS
        nn
        metalium
    GROUP ml
    DESCRIPTION "TT-Train runtime library"
)
cpack_add_component(ttml GROUP ml)
```